src/cpp/core/r_util/RToolsInfo.cpp

```diff
@@ -296,6 +299,69 @@ Error scanRegistryForRTools(bool usingMingwGcc49,
       return Success();
 }
 
+Error scanFoldersForRTools(bool usingMingwGcc49, std::vector<RToolsInfo>* pRTools)
+{
+   // look for Rtools as installed by RStudio
+   std::string systemDrive = core::system::getenv("SYSTEMDRIVE");
+   FilePath buildDirRoot(systemDrive + "/RBuildTools");
+
+   // find sub-directories
+   std::vector<FilePath> buildDirs;
+   Error error = buildDirRoot.children(&buildDirs);
+   if (error)
+      LOG_ERROR(error);
```

Comment:
I think this will log an error for all users who don't have a `%SYSTEMDRIVE%\RBuildTools` directory, which might be noisy since lots of windows users won't (although I can't remember offhand if we scan for R versions as a matter of course). Maybe we should succeed silently here if `buildDirRoot` doesn't exist?

Reply:
You're right; we should definitely check that the directory exists first. I'll make that change!